python/dlisio/lis/load.py

```diff
@@ -42,6 +42,7 @@ def load(path, error_handler = None):
         try:
             indexer.index_logical_file()
         except:
+            indexer.close()
```

Comment:
Why is this a separate commit?

Reply:
It was supposed to be a separate commit because the first commit fails because I am not closing enough stuff (and I am no longer sure what, probably all). Then in second commit I refactor that thing. So in third commit I just wanted to explicitly fix the thing from first (probably) by closing everything. Might not be that.

But will merge commits 2 and 3.